src/scripts/cfo.zig

```diff
@@ -58,6 +58,9 @@ pub const CliArgs = struct {
     budget_minutes: u64 = 10,
     hang_minutes: u64 = 30,
     concurrency: ?u32 = null,
+    // Don't checkout a separate repository, and instead fuzz in-place. Useful for debugging
+    // CFO itself.
+    no_checkout: bool = false,
```

Comment:
Alternatively, we could let `gh_token == null` imply `no_checkout`.
Since if `gh_token == null` we aren't downloading extra branches, either.

Reply:
Excellent suggestion, thanks!